packages/backend/src/graphql/resolvers/wallet_address.test.ts

```diff
@@ -35,21 +35,27 @@ import {
 import { getPageTests } from './page.test'
```

Comment:
Should there be tests to check if operators can perform wallet address mutations for other tenants? I recall debating whether or not if operators creating wallet addresses for tenants was a case we felt was worth handling, but maybe for the other operations?

Reply:
@njlie By default. the `beforeAll` will ensure that the `isOperator` is `false`. 
I added the following tests to evaluate the above:
- `bad input data when not allowed to perform cross tenant create`
- `bad input data when not allowed to perform cross tenant update`
- `Can get page of wallet addresses with tenantId param`
- `Can get a wallet address` updated to ensure the NotFound error is thrown
Which I think is extensive enough, due to the same `tenantIdToProceed` mechanism being used.

I am going to resolve this for now, but if you think we need more, please re-open and ping me.